tests/stdlib/trst.nim

```diff
@@ -439,12 +439,7 @@ suite "RST inline markup":
           rnLeaf  ' '
           rnLeaf  '('
           rnStandaloneHyperlink
-            rnLeaf  'http'
-            rnLeaf  ':'
-            rnLeaf  '//'
-            rnLeaf  'no'
-            rnLeaf  '.'
-            rnLeaf  'org'
+            rnLeaf  'http://no.org'
```

Comment:
in this or followup PR, can you add tests with more complicated urls, involving ?, & etc ?

Reply:
Added some arbitrary link https://www.google.com/url?sa=t&source=web&cd=&cad=rja&url=https%3A%2F%2Fnim-lang.github.io%2FNim%2Frst.html%23features&usg=AO&= 

Addition of final `&=' is invented to check the current behavior. Previously it would include `&` but not final `=`. Now it skips both `&=`.